### PR TITLE
Fix Voice Call Doc API Resolves: OKTA-109039

### PR DESCRIPTION
**Pull request by gauthamgowrishankar-okta**
This change will fix the unsupported update="true" for Voice Call MFA Factor

### CODE REVIEW

File: _source/_docs/api/resources/factors.md [hunk 1]
```diff
@@ -755,18 +755,6 @@ curl -v -X GET \
...
-          {
-            "id": "mblldntFJevYKbyQQ0g4",
-            "profile": {
-              "phoneNumber": "+14081234567",
-              "phoneExtension": "1234"
-            },
-            "status": "ACTIVE"
-          }
-        ]
       }
```
Comment (mystiberry-okta) [inline _source/_docs/api/resources/factors.md]: @gauthamgowrishankar-okta  should line 758 also be deleted?

Comment (gauthamgowrishankar-okta) [inline _source/_docs/api/resources/factors.md]: yes

Comment (gauthamgowrishankar-okta) [inline _source/_docs/api/resources/factors.md]: screen shot 2016-11-30 at 1 00 18 pm

Comment (gauthamgowrishankar-okta) [inline _source/_docs/api/resources/factors.md]: the identation looks fine

**Comment by mystiberry-okta** [comment]
@gauthamgowrishankar-okta Okay to merge?

**Comment by gauthamgowrishankar-okta** [comment]
@srinivasanagandla-okta  dos this change look okay ?

**Comment by srinivasanagandla-okta** [comment]
Looks good. 🚢

**Pull request merged**